### PR TITLE
🎨 List question block redesign block redesign #626

### LINKED ISSUE #626: :art:  List Question Block Redesign: block redesign
## User stories

As a user, I want the "List Block" component in the app to be redesigned to enhance its usability and visual appeal.

---

Quick links: [Summary](#summary) | [Description](#description) | [Template](#template) | [Example](#example) | [Resources](#resources)

---

### Technical Description
The current design is: 
image

It should be changed to:
image

The files can be found at `libs/features/convs-mgr/stories/blocks/library/list-message-block/`. 

The designs are [here](https://www.figma.com/file/ZGHzSstXOYPMFU1EiqjEHD/Untitled?type=design&node-id=1%3A4&mode=design&t=Hf7DxaUUMQ3BWy57-1)

### Acceptance Criteria

1. The block should look exactly like the designs.
2. Use of `rem`, `em` as opposed to `px`.
3. All changes should be isolated to the following  `libs/features/convs-mgr/stories/blocks/library/list-message-block/`.

### PR DESCRIPTION
**Pull request by Mutagah**
# Description

This PR updates the UI of the `question list` block.

Fixes #626

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# Screenshot (optional)

updated_question_list

updated_question_list3

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: libs/features/convs-mgr/stories/blocks/library/main/src/lib/providers/list-block.jsplumb.ts [hunk 1]
```diff
@@ -21,9 +21,8 @@ export function _ListBlockDecoratePlumb(block: ListMessageBlock, comp: Component
     // Whether the anchor is target (Other Block -> This Block)
     target: true,
     cssClass:"block_endpoint",
-
     endpoint: 'Dot',
-    anchor: "Left",
+    anchor: [0, 0.10, 0, 1],
```
Comment (LemmyMwaura) [inline libs/features/convs-mgr/stories/blocks/library/main/src/lib/providers/list-block.jsplumb.ts:25]: Lets leave this as 'Left'

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
Sync your changes then request a review

File: libs/features/convs-mgr/stories/blocks/library/main/src/lib/providers/list-block.jsplumb.ts (hunk 1 above)
Comment (Mutagah) [reply libs/features/convs-mgr/stories/blocks/library/main/src/lib/providers/list-block.jsplumb.ts]: > Lets leave this as 'Left'

Changes requested have been addresses

**Review by LemmyMwaura — APPROVED** [review]
👍 awesome work

**Pull request merged**